pages/linux/script.md

```diff
@@ -0,0 +1,19 @@
+# script
+
+> Record all terminal output to file.
+
+- Record a new session to a file named `typescript` in the current directory:
+
+`script`
+
+- Record a new session to custom filepath:
+
+`script {{path/to/session.out}}`
+
+- Record a new session, appending to an existing file:
+
+`script -a session.out`
```

Comment:
Is session.out a file? Can the name change?

Reply:
Looks like it is, I'd suggest adding the `{{parentheses}}` around it.

Reply:
Yes. That was my concern.

Reply:
@mebeim and @ schneiderl I hear and obey. I also changed the timing file name for consistency.